### ISSUE TITLE
Source code is not really ES5 (UglifyJS error)

### ISSUE THREAD

**Issue by khakulov** [jossmac/react-scrolllock#1]
```javascript
{
	componentWillMount() { ... }
}
```

is not ES5!!! It should be

```javascript
{
	componentWillMount: function() { ... }
}
```

**Comment by jossmac** [comment]
Ref #104

**Comment by jossmac** [comment]
Fixed by be609e3c3829e5f4d9cdb9a332a95c922f35ada3
